pccs/setup.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e
+
+mkdir -p /opt/intel/pccs/config/
```

Comment:
```suggestion
```
The following code assumes that /opt/intel/pccs/config/default.json exists, so this is superfluous, right?

Reply:
Yes, it is not needed.